_posts/2012-07-10-on-demand-jenkins-slaves-with-amazon-ec2.markdown

```diff
@@ -1,55 +1,80 @@
 ---
 layout: post
-title: "On-Demand Jenkins Slaves with Amazon EC2"
+title: "On-Demand Jenkins Agents with Amazon EC2"
```

Comment:
Note that I did not change the filename of this article, because I don't know of a way to do redirects in Jekyll/github pages. If anyone is more familiar & has a suggestion on how we can do that, it'd be great to clean up the path to this article.